js-wether/wether.js

```diff
@@ -0,0 +1,96 @@
+document.addEventListener("DOMContentLoaded", function () {
+    //指定されたプルダウンの値を取得
+    let getSelectValue = function (name) {
+        let result = "";
+        let elems = document.getElementById(name).options;
+        for (let i = 0, len = elems.length; i < len; i++) {
+            let elem = elems.item(i);
+            if (elem.selected) {
+                result = elem.value;
+                break;
+            }
+        }
+        return result;
+    };
+
+    //APIへの問い合わせ～HTML反映まで
+    let getAPI = function () {
+        //APIへの問い合わせURLを生成
+        let url = "http://api.openweathermap.org/data/2.5/weather?q=" + getSelectValue("city") + "&APPID=4b5774e9f3d2a07b84f0f2f88e486224";
```

Comment:
Java等の言語でも同じなのですが、`+`による文字列作成はあまり好ましくないので、極力さけると良いです。

理由としては2点あります。
後者は言語によっては関係ない場合もありますし、この程度のコードであればそこまで関係ないです。（知識として知っておくといいかなぁーという感じ）

- `+` 前後でコードが読みにくい（どういった文字列かわかりにくくなりやすい）
- `+` で文字列を足す処理は、ちょっとパフォーマンスが悪い（※1）

※1 
```
Java（JavaScriptではない）の説明ですが、雰囲気だけ掴んでもらえると
https://qiita.com/nkojima/items/0098dccbe4a593bc0306#%E3%81%AB%E3%82%88%E3%82%8B%E7%B5%90%E5%90%88
```

javascriptのテンプレートリテラルを使うとスッキリかけます

```suggestion
        //APIへの問い合わせURLを生成
        const cityName = getSelectValue("city");
        const appId = "4b5774e9f3d2a07b84f0f2f88e486224";

        let url = `http://api.openweathermap.org/data/2.5/weather?q=${cityName}&APPID=${appId}`;
```